cmd/oadp/backup.go

```diff
@@ -57,13 +57,15 @@ https://hypershift.pages.dev/how-to/disaster-recovery/dr-cli/`,
 	cmd.Flags().StringVar(&opts.HCNamespace, "hc-namespace", "", "Namespace of the hosted cluster to backup (required)")
 
 	// Optional flags with defaults
+	cmd.Flags().StringVar(&opts.BackupCustomName, "name", "", "Custom name for the backup (must be 63 characters or less, auto-generated if not specified)")
 	cmd.Flags().StringVar(&opts.OADPNamespace, "oadp-namespace", "openshift-adp", "Namespace where OADP operator is installed")
 	cmd.Flags().StringVar(&opts.StorageLocation, "storage-location", "default", "Storage location for the backup")
 	cmd.Flags().DurationVar(&opts.TTL, "ttl", 2*time.Hour, "Time to live for the backup")
 	cmd.Flags().BoolVar(&opts.SnapshotMoveData, "snapshot-move-data", true, "Enable snapshot move data feature")
 	cmd.Flags().BoolVar(&opts.DefaultVolumesToFsBackup, "default-volumes-to-fs-backup", false, "Use filesystem backup for volumes by default")
 	cmd.Flags().BoolVar(&opts.Render, "render", false, "Render the backup object to STDOUT instead of creating it")
 	cmd.Flags().StringSliceVar(&opts.IncludedResources, "included-resources", nil, "Comma-separated list of resources to include in backup (overrides defaults)")
+	cmd.Flags().StringSliceVar(&opts.IncludeNamespaces, "included-namespaces", nil, "Override included namespaces (by default includes hc-namespace and hc-namespace-hc-name)")
```

Comment:
s/hc-namespace-hc-name/control-plane-namespace/

Also, just curious, under which circumstances would we specify something other than the hc and hcp namespaces?

Reply:
In baremetal we also need to backup the infraenv and the postgres DDBB from multicluster-engine namespace in order to have a suitable restoration in other MGMT cluster (when this got supported). Also the restoration object required the backup object to be present in the destination MGMT cluster so, eventually we will want that too.